.travis.yml

```diff
@@ -6,7 +6,7 @@ rvm:
   - 2.2.6
   - jruby-9.0.5.0
 gemfile:
-  - gemfiles/ar50.gemfile
+  - gemfiles/ar51.gemfile
```

Comment:
If ActiveRecord has changed a lot internally, I like this style (a major version bump to the adapter and maintaining separate git & release branches of the adapter, corresponding to AR 5.0 & 5.1). If it's a minor change, I'll support both AR versions in a single branch.